src/no/runsafe/framework/api/entity/animals/horses/IAbstractHorse.java

```diff
@@ -0,0 +1,19 @@
+package no.runsafe.framework.api.entity.animals.horses;
+
+import no.runsafe.framework.api.entity.IAnimal;
+import no.runsafe.framework.api.entity.ITameable;
+
+public interface IAbstractHorse extends IAnimal, ITameable
```

Comment:
I see an IHorse and an IChestedHorse, but I don't see any usage of the chested one, and Abstract makes for a poor name. I see no reason to reflect poorly named classes upstream when we can do better.